package.json

```diff
@@ -71,7 +73,7 @@
     "postcss": "8.4.21",
     "prettier": "2.8.4",
     "process": "^0.11.10",
-    "prop-types": "^15.7.2",
+    "prop-types": ">=15.0.0",
```

Comment:
you can just remove this one, we are not using prop-types after migration to typescript, I just forgot to remove this guy, sorry, my fault!